subgraph.template.yaml

```diff
@@ -37,10 +37,6 @@ dataSources:
           handler: handleThirdPartyItemSlotsBought
         - event: ThirdPartyReviewedWithRoot(string,bytes32,bool,address)
           handler: handleThirdPartyReviewedWithRoot
-        - event: ItemAdded(string,string,string,bool,address)
-          handler: handleItemAdded
-        - event: ItemUpdated(string,string,string,address)
-          handler: handleItemUpdated
```

Comment:
Don't we need to remove `ItemReviewed` as well?
Should we delete the item entity too?
```
      entities:
        - ThirdParty
        - Item
```